src/main/java/reposense/parser/RepoConfigCsvParser.java

```diff
@@ -22,9 +25,17 @@
     private static final int IGNORE_GLOB_LIST_POSITION = 3;
     private static final int IGNORE_STANDALONE_CONFIG_POSITION = 4;
     private static final int IGNORE_COMMIT_LIST_CONFIG_POSITION = 5;
+    private static final int GROUPS_CONFIG_POSITION = 6;
+
+    private static String configFolderPath;
 
     public RepoConfigCsvParser(Path csvFilePath) throws IOException {
         super(csvFilePath);
+        if (csvFilePath.getParent() != null) {
+            this.configFolderPath = csvFilePath.getParent().toString() + '/';
+        } else {
+            this.configFolderPath = "./";
```

Comment:
Why is this change necessary?

Also, this is not a very clean way to do it, seems rather OS dependent. Can you please do it the same way as how the `-config` is parsed?

Reply:
I need the ```configFolderpath``` in order to retrieve the ```groups-config.csv``` file, which contains the groupings specification, and pass it to ```GroupConfigCsvParser```.

Reply:
Sorry, I'm not sure how to do it as ```-config``` is parsed since it makes use of ```Argument``` and ```ArgumentParser```. Is there a way I can retrieve the configFolder parsed in the arguments and use it here?

Reply:
>Sorry, I'm not sure how to do it as -config is parsed since it makes use of Argument and ArgumentParser. Is there a way I can retrieve the configFolder parsed in the arguments and use it here?

There are several ways to go about doing that, I leave it to you to figure it out as I do not have much time recently to launch my development pc. 
For example, your `groups-config.csv` needs to be on the same folder as `repo-config.csv` and both of them use fixed names. So why don't you just pass in the `configFolderPath` in the first case?

Reply:
> For example, your `groups-config.csv` needs to be on the same folder as `repo-config.csv` and both of them use fixed names. So why don't you just pass in the `configFolderPath` in the first case?

The problem is, ```groups-config.csv``` is not a fixed name since there can be multiple ```groups-config.csv``` for each repository. For instance, for ```reposense-Alpha```, the filename can be ```alpha-groups-config.csv```, and for ```reposense-Delta```, the filename can be ```delta-groups-config.csv```.

Reply:
Mistake on my part of comment there, but u only need the folder path don't you?
In that case, since `repo-config.csv` is fixed, you can do so.